dashboard/models.py

```diff
@@ -0,0 +1,40 @@
+"""
+Models for dashboard
+"""
+
+from django.contrib.auth.models import User
+from django.db.models import (
+    DateTimeField,
+    ForeignKey,
+    Model,
+    OneToOneField,
+)
+from jsonfield import JSONField
+
+from courses.models import CourseRun
+
+
+class Enrollment(Model):
+    """
+    Model for user enrollment data from edX
+    """
+    user = ForeignKey(User)
+    course_run = ForeignKey(CourseRun)
+    data = JSONField(null=True)
+    last_request = DateTimeField()
+
+    class Meta:
+        unique_together = (('user', 'course_run'), )
+
+
+class Certificate(Model):
```

Comment:
what do you think about renaming these to `CachedEnrollment` and `CachedCertificate`? more accurately describes the data, and it would avoid naming collisions with `edx_api` classes